perceval/components/abstract_processor.py

```diff
@@ -261,14 +261,17 @@ def _validate_postselect_composition(self, mode_mapping: Dict):
                 f"Post-selection conditions cannot compose with modes {impacted_modes}"
 
     def _compose_processor(self, connector: ModeConnector, processor, keep_port: bool):
+        logger.debug(f"Compose processor {self.name} with {processor.name}", channel.general)
         self._is_unitary = self._is_unitary and processor._is_unitary
         self._has_td = self._has_td or processor._has_td
         if processor.heralds and not processor.parameters:
             # adding the same processor component again renders incorrect heralds if not copied
             # This concerns our gate based processors from catalog which has no input params
+            logger.debug("  Force copy during processor compose", channel.general)
```

Comment:
Extra space at the beginning of the message. It seems intentional for readability - something like an indent for subsequent debug messages inside. I am wondering if there is any other way to implement this instead of typing the 'space' ?

Reply:
I don't think so, some function will have to eventually add these spaces or tab symbols.

Reply:
We could implement an "indent" parameter (as a positive integer) but I'd say it's at least another task, and I'm not so sure we'd need it.